### PR TITLE
correct typos in comments

### PR DESCRIPTION
**Pull request by AJFSalomon**
# Context
Yup, I know, this is a futile PR. I was bored after work, and after my last PR (#7166) I thought I would try to find more typos in the repo. These are clearly not important changes so I don't mind if the PR gets discarded.

# Description
I've looked for typos in comments and literals, and tried to fix everything I could find. I'm not 100% sure of all my corrections, so even if it's just comments, I'd recommend that other contributors check what I've done before merging. However, I did not look for typos in the actual code. That's a whole other daunting task I didn't have time for, sorry.

### CODE REVIEW

File: napari/utils/colormaps/vendored/cm.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 def _reverser(f, x=None):
     """Helper such that ``_reverser(f)(x) == f(1 - x)``."""
     if x is None:
-        # Returning a partial object keeps it picklable.
+        # Returning a partial object keeps it pickable.
```
Comment (psobolewskiPhD) [inline napari/utils/colormaps/vendored/cm.py]:
```suggestion
        # Returning a partial object keeps it pickleable.
```
Pretty sure that's what was intended here
https://en.wiktionary.org/wiki/pickleable

Comment (AJFSalomon) [reply napari/utils/colormaps/vendored/cm.py]: You're probably right, I didn’t know this word. I'll do an edit ASAP.

**Review by psobolewskiPhD — CHANGES_REQUESTED** [review]
Thanks!
Definitely nice to clean that stuff up.
I did catch one fix that is not correct.

**Comment by psobolewskiPhD** [comment]
We could consider something like: https://github.com/crate-ci/typos
> typos maintains a list of known typo corrections to keep the false positive count low so it can safely run unassisted.

**Comment by AJFSalomon** [comment]
Thanks for correcting me ! I commited the new change 😁

**Review by jni — APPROVED** [review]
> These are clearly not important changes so I don't mind if the PR gets discarded.

Not at all! Actually I often come across one of these while fixing something else and think, "ugh, I should fix that later" and then never get around to it! 😂 So actually I am very grateful for how expansive this PR is. 😂 🙏 Thanks!

**Review by psobolewskiPhD — APPROVED** [review]

**Pull request merged**